persistence-jdbc/javadsl/src/main/scala/com/lightbend/lagom/internal/persistence/jdbc/JdbcOffsetStore.scala

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package com.lightbend.lagom.internal.persistence.jdbc
```

Comment:
the package name must be changed for all internal, so that we don't use the same package name in two jars
```
com.lightbend.lagom.internal.javadsl.persistence.jdbc
```